buzsaki_lab_to_nwb/valero/eventsinterface.py

```diff
@@ -0,0 +1,211 @@
+        self.session_path = Path(self.source_data["folder_path"])
+        self.session_id = self.session_path.stem
+
+        # We use the behavioral cellinfo file to get the trial intervals
+        hse_data_path = self.session_path / f"{self.session_id}.HSE.mat"
+        if not hse_data_path.exists():
+            warnings.warn(f"HSE event file not found: {hse_data_path}. Skipping HSE events interface.")
+            return nwbfile
+
+        mat_file = read_mat(hse_data_path, variable_names=["HSE"])
+        hse_data = mat_file["HSE"]
+
+        hse_intervals = hse_data["timestamps"]
+        peaks = hse_data["peaks"]
+        center = hse_data["center"]
+
+        column_descriptions = dict(
+            peaks="TBD",
+            center="TBD",
+        )
+
+        column_data_dict = dict(
+            peaks=peaks,
+            center=center,
+        )
+
+        name = "HSETimeIntervals"
+        description = "TBD"  # TODO: Ask author for description
+        ripple_events_table = TimeIntervals(name=name, description=description)
+
+        for start_time, stop_time in hse_intervals:
+            ripple_events_table.add_row(start_time=start_time, stop_time=stop_time)
+
+        for column_name, column_data in column_data_dict.items():
+            ripple_events_table.add_column(
+                name=column_name,
+                description=column_descriptions[column_name],
+                data=H5DataIO(column_data, compression="gzip"),
+            )
+
+        processing_module = get_module(nwbfile=nwbfile, name="ecephys")
+
+        processing_module.add(ripple_events_table)
+
+
+class ValeroRipplesEventsInterface(BaseDataInterface):
+    def __init__(self, folder_path: FolderPathType):
+        super().__init__(folder_path=folder_path)
+
+    def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = False):
+        self.session_path = Path(self.source_data["folder_path"])
+        self.session_id = self.session_path.stem
+
+        # We use the behavioral cellinfo file to get the trial intervals
+        ripples_file_path = self.session_path / f"{self.session_id}.ripples.events.mat"
+        assert ripples_file_path.exists(), f"Ripples event file not found: {ripples_file_path}"
+
+        mat_file = read_mat(ripples_file_path, variable_names=["ripples"])
+        ripples_data = mat_file["ripples"]
+
+        ripple_intervals = ripples_data["timestamps"]
+        if ripple_intervals.size == 0:
+            warnings.warn(f"No ripples found for session: {self.session_id}. Skipping ripple events interface")
+            return nwbfile
+
+        # Name and descriptions
+        field_in_mat_file_to_nwb_column_info = dict()
+        if "peaks" in ripples_data and ripples_data["peaks"].size != 0:
+            field_in_mat_file_to_nwb_column_info["peaks"] = dict(
+                name="peaks", description="Peak of the ripple.", data=ripples_data["peaks"]
+            )
```

Comment:
From looking at `fCamk2_201012_sess1`, the data for `peaks` actually appears to be the _time_ at which the peak occurs, not the value of the peak (which I would have thought from the name)

As such, can we call this column `peak_time`? (and adjust description)

Reply:
Thanks for noticing that. I actually have peakAmpltidue as well so I was aware but the description and the convention of using _time is indeed missing.